src/transformers/trainer.py

```diff
@@ -646,6 +686,11 @@ def log(self, logs: Dict[str, float], iterator: Optional[tqdm] = None) -> None:
         if is_wandb_available():
             if self.is_world_process_zero():
                 wandb.log(logs, step=self.global_step)
+        if is_comet_available():
+            if self.is_world_master():
```

Comment:
```suggestion
            if self.is_world_process_zero():
```
(`is_world_master` is now deprecated)